### ISSUE TITLE
Documentation for CategorySchemaManager.assign_to_items() is incorrect

### ISSUE THREAD

**Issue by JonathanBaileyTC** [Esri/arcgis-python-api#571]
The documentation that describes the schema of the array of JSON objects to pass to the CategorySchemaManager.assign_to_items() method is incorrect. It indicates that the schema should be:

```
[{
    “2678d3002eea4e4a825e3bdf10016e61”: {
        “categories”: [“/Categories/Geology”, “/Categories/Elevation”]
    }
}, {
    “c3ad4ed8bcf04d619537cfe252a1760d”: {
        “categories”: [“/Categories/Geology”, “/Categories/Land cover/Forest/Deciduous Forest”]
    }
}, {
    “9ced00fdce3e4b20bb4b05155acbe817”: {
        “categories”: []
    }
}]
```

when in fact it should be:

```
[{
    “2678d3002eea4e4a825e3bdf10016e61”: [“/Categories/Geology”, “/Categories/Elevation”]
},
{
    “c3ad4ed8bcf04d619537cfe252a1760d”: [“/Categories/Geology”, “/Categories/Land cover/Forest/Deciduous Forest”]
},
{
    “9ced00fdce3e4b20bb4b05155acbe817”: []
}]
```

**Comment by CMPeng** [comment]
@JonathanBaileyTC 
Can you please try the workflow again? With API version at 1.7.1, the `items` input has to be exactly the form as what is stated in the help doc, e.g.
```
a_csm = a_gis.content.categories
a_csm.assign_to_items([{
"cff5bfca2903400eb7320d9717206402": {
"categories": ["/Categories/Geology4", "/Categories/Elevation4"]

}

}, {
"75502c56faac4c64b54809f39a8f5d08": {
"categories": ["/Categories/Geology", "/Categories/Land cover/Forest/Deciduous Forest"]

}

}, {
"bd7044be0d144c358787095f27191eae": {
"categories": []

}}
])
```
or else, the returned list of status would be:

```
{'75502c56faac4c64b54809f39a8f5d08': False,
 'bd7044be0d144c358787095f27191eae': False,
 'cff5bfca2903400eb7320d9717206402': False}
```

**Comment by jholycross** [comment]
@CMPeng Working today with AGOL, I'm observing the same behavior that Jonathan mentioned. When I use the assign_to_items input as documented in the help, all of my returned statuses are false.  When I use the format that he mentioned (and is the same as the REST API updateCategories), my changes are successful and return True.

**Comment by CMPeng** [comment]
@jholycross Can you please check if you have the latest ArcGIS API for python? By running the codes below:
```
import arcgis
arcgis.__version__
```
There had been a bug fix in April 2019, which should support reading in the categories in dict object since then (1.7). Please confirm that you have installed version 1.7 or above. As of today, the latest version is 1.8.

**Comment by achapkowski** [comment]
This is resolved and verified at v1.8.1
